Add apple.txt with a description of apples

sabatjohn17 opened the pull request.

sabatjohn17 committed:
Add apple.txt with a description of apples

ajit-spec reviewed (commented):
add few more lines

ajit-spec reviewed (commented):
also add 1 more file named orange.txt with few details

sabatjohn17 committed:
Add nutritional information to apple.txt and create orange.txt

- Updated apple.txt to include information about dietary fiber and
vitamin C.
- Added a new file, orange.txt, describing the nutritional benefits of
oranges.

ajit-spec reviewed (approved):
good job, approved.
